util/embedders.py

```diff
@@ -35,11 +40,21 @@ def get_embedder(
                 embedder = TfidfSentenceEmbedder(batch_size=batch_size)
             else:
                 raise Exception(f"Unknown model {model}")
-        elif platform == enums.EmbeddingPlatform.OPENAI.value:
+        elif (
+            platform == enums.EmbeddingPlatform.OPENAI.value
+            or platform == enums.EmbeddingPlatform.AZURE.value
+        ):
+            if platform == enums.EmbeddingPlatform.AZURE.value:
+                batch_size = 16
+                model = "azure-openai-embeddings"
```

Comment:
if this is fixed do we still need it in the UI then?

Reply:
Removed the model and the type from the UI, since they are fixed (l left the model name in the UI as a default so the name appears in the notification, and the user would know which model is used)